### PR TITLE
:sparkles: error message on input

### PR DESCRIPTION
**Pull request by guillaumecrespel**

### CODE REVIEW

File: src/input/__snapshots__/input.spec.js.snap [hunk 1]
```diff
@@ -379,6 +450,9 @@ exports[`common/Input graphical (JSX) types should print an input (checkbox) 1`]
   >
     {"name":"checkbox","props":{"input":{"name":"name","value":"","checked":false},"meta":{"active":false,"asyncValidating":false,"autofilled":false,"dirty":false,"form":"testForm","invalid":false,"pristine":true,"submitting":false,"submitFailed":false,"touched":false,"valid":true,"visited":false},"className":"","placeholder":"","disabled":false,"type":"checkbox","children":null}}
   </div>
+  <div
```
Comment (fabienjuif) [inline src/input/__snapshots__/input.spec.js.snap]: The error message div is always printed ? 
This is intended ?

File: src/input/input.styles.scss [hunk 2]
```diff
@@ -25,6 +25,18 @@
     }
   }
 
+  .msg {
```
Comment (fabienjuif) [inline src/input/input.styles.scss]: I would rename the classe `errorMsg` or something else.
But not `msg` alone, as this doesn't evoke the error nature of the message.

File: src/input/__snapshots__/input.spec.js.snap (hunk 1 above)
Comment (guillaumecrespel) [reply src/input/__snapshots__/input.spec.js.snap]: yes fot opacity effect, like a label

File: src/input/input.styles.scss [hunk 3]
```diff
@@ -148,7 +160,13 @@
   }
 }
 
+.error ~ .msg {
+  opacity: 1;
```
Comment (fabienjuif) [inline src/input/input.styles.scss]: We quickly speak about it.
Opacity transition are not good for mobile.

File: src/input/__snapshots__/input.spec.js.snap (hunk 1 above)
Comment (fabienjuif) [reply src/input/__snapshots__/input.spec.js.snap]: Ok I understand that you play with opacity

File: src/input/input.styles.scss (hunk 2 above)
Comment (guillaumecrespel) [reply src/input/input.styles.scss]: :+1:

File: src/input/input.container.js [hunk 4]
```diff
@@ -1,15 +1,15 @@
 import { connect } from 'react-redux'
-import { change, formValueSelector, getFormSubmitErrors } from 'redux-form'
+import { change, formValueSelector, getFormSyncErrors } from 'redux-form'
 import { formInjector } from '../helpers'
 import Component from './input'
 
 const mapStateToProps = (state, { form, name, placeholder, label, withoutLabel }) => {
-  const submitErrors = getFormSubmitErrors(form)(state)
+  const error = getFormSyncErrors(form)(state)
```
Comment (fabienjuif) [inline src/input/input.container.js]: Can you explain me why this is better here to use `getFormSyncErrors` ?

Comment (guillaumecrespel) [reply src/input/input.container.js]: `getFormSubmitErrors` get error after submit only

File: src/input/input.styles.scss (hunk 3 above)
Comment (guillaumecrespel) [reply src/input/input.styles.scss]: it's the same problem for label, but, for the moment, i've no idea for sexy UI whithout this?
Translation ?

Comment (fabienjuif) [reply src/input/input.styles.scss]: Let it this way right now, I don't know how to do this fine.

 - Maybe adding a `transform: scale(1)` would do the trick (it force the navigator to use GPU)
 - This is worth doing some benchmark one day

**Pull request merged**